Dockerfile

```diff
@@ -12,6 +12,11 @@ RUN yum install -y $INSTALL_PKGS && \
     yum clean all && \
     rm -rf /tmp/*
 
+# Install jq : command-line JSON processor
+RUN curl -L https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64 > jq && \
```

Comment:
@Kusanagi9999 -- I think for the time being if we can remove the use of jq / including the jq binary here, we could be OK to get this merged into master. Granted, it'll make the resulting config a little less human legible, but... I think that'll be OK, it'll still parse. One of my colleagues mentioned that idea of making a go application to pretty print JSON and doing a multi-stage build, but... I think that might be overkill for now.

Thank you a bunch for the pull request -- cool feature!

Reply:
OK, sounds good. I removed jq for now.